python-qa-tools/README.md

```diff
@@ -0,0 +1,106 @@
+This repo contains python scripts which can be used for quickly setting up a local test environment for any Cosmos based network with *n* number of nodes and *m* number of accounts. It installs **go** if it's not already installed on the system and also installs all the dependencies along with it.
```

Comment:
is this still valid?

Reply:
Yeah, I am going to remove the entire `README.md` file which is there in the `python-qa-tools` folder. Instead I am going to rewrite the same instructions in Global `README.md` file.

Reply:
I have removed the README docs inside the python-qa-tools module.
Global README is there.